index.html

```diff
@@ -4,288 +4,286 @@
+                          <li class="li-tag-list btn-tag">Ruby on rails</li>
+                          <li class="li-tag-list btn-tag">RSpec</li>
+                       </ul>
+                       <ul class="uld-tags">
+                           <li class="li-tag-list btn-tag">Capybara</li>
+                           <li class="li-tag-list btn-tag">Selenium</li>
+                       </ul>
+                   </div>
+               </div>
+           </div>
+           <div class="cards-work">
+               <div class="figcard">
+                   <figure>
+                       <img src="./images/IconThreeDesktop.png" alt="Icon">
+                   </figure>
+               </div>
+               <div class="languaged">
+                   <div class="title">
+                       <h4>Skills</h4>
+                   </div>
+                   <div class="lang-list">
+                       <ul class="uld-tags">
+                           <li class="li-tag-list btn-tag">Codekit</li>
+                           <li class="li-tag-list btn-tag">GitHub</li>
+                           <li class="li-tag-list btn-tag">Codepen</li>
+                       </ul>
+                       <ul class="uld-tags">
+                           <li class="li-tag-list btn-tag">Gitlab</li>
+                           <li class="li-tag-list btn-tag">Terminal</li>
+                       </ul>
+                   </div>
+               </div>
+           </div>
+       </div>
+   </section>  
+   <section class="form">
+       <div class="info">
+           <p>
+               I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+           </p>
+       </div>
+       <div class="form-details">
+           <form action="https://formspree.io/f/xeqnbggn" method="post" class="formd">
+               <input type="text" placeholder="Full Name" class="name">
+               <input type="email" placeholder="Email address" class="email">
+               <textarea placeholder="Type something"></textarea>
+               <button class="btn-primary">Get in touch</button>
+           </form>
+       </div>  
+   </section> 
+<section class="footer">
+       <div class="footer-image">
+           <figure>
+               <img src="./images/bottomleft.svg" alt="imgleft" class="imgleft">
+               <img src="./images/bottomiddl.svg" alt="imgmiddle" class="imgmiddle">
+               <img src="./images/bottomright.svg" alt="imgright" class="imgright">
+           </figure>
+       </div>
+       <hr class="bottom-line">
+       <div class="medias">
+           <ul class="ul-socialmedia">
+               <li class="socialists"><a href="https://twitter.com/aidan-bn"><img src="./images/twitter.svg" alt="twitter"></a></li>
+               <li class="socialists"><a href="https://twitter.com/aidan-bn"><img src="./images/linkedin.svg" alt="linkedin"></a></li>
```

Comment:
- [ ] Same thing as before for the name of the LinkedIn file, kindly fix this so it locates the right file. 😁

Reply:
Resolved